### PR TITLE
Added DB migration

### PR DESCRIPTION
**Pull request by siarhei-shliayonkin**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,6 +4,17 @@ VERSION_MOCKGEN=v1.6.0
...
 RELEASE_NAME_APP=app
 
+# helm chart path
+HELM_CHART_DB = helm-charts/database-to-integrity-sum
+HELM_CHART_APP = helm-charts/app-to-monitor
+
+# DB params
+DB_NAME	?= $(PGDATABASE)
+DB_USER	?= $(PGUSER)
+DB_PASSWORD	?= $(PGPASSWORD)
```
Comment (alavrovinfb) [inline Makefile]: Setting `PGPASSWORD` and other DB envs is unnecessary, you can just export `DB_PASSWORD`

File: README.md [hunk 2]
```diff
@@ -94,11 +94,14 @@ If you want to generate binaries for different platforms:
 make compile
 ```
 ### Manual start
-Set some values
-+ set the value `secretNameDB` in the file `helm-charts/app-to-monitor/values.yaml` to be the same as the value of `secretName` in the file `helm-charts/database-to-integrity-sum/values.yaml`
-+ set the value `releaseNameDB` in the file `helm-charts/app-to-monitor/values.yaml` to be the same as the release name you will use when installing the database
+Set some environment variables to configure DB params:
+- PGPASSWORD
```
Comment (alavrovinfb) [inline README.md]: Setting `PGPASSWORD` and other DB envs is unnecessary, you can just export `DB_PASSWORD`

**Review by Kaborda-Irina-ScienceSoft — APPROVED** [review]
Looks good

File: Makefile (hunk 1 above)
Comment (siarhei-shliayonkin) [reply Makefile]: done

**Review by alavrovinfb — APPROVED** [review]
looks good

File: README.md (hunk 2 above)
Comment (siarhei-shliayonkin) [reply README.md]: It's just a reminder for a user.

**Pull request merged**